agent/taskresource/credentialspec/credentialspec_windows.go

```diff
@@ -529,9 +529,15 @@ func (cs *CredentialSpecResource) clearCredentialSpec() {
 			seelog.Debugf("Skipping cleanup of local credentialspec file option: %s", key)
 			continue
 		}
-		credSpecSplit := strings.SplitAfterN(value, "credentialspec=", 2)
+		// Split credentialspec to obtain local file-name
+		credSpecSplit := strings.SplitAfterN(value, "credentialspec=file://", 2)
+		if len(credSpecSplit) != 2 {
+			seelog.Warnf("Unable to parse target credentialspec: %s", value)
+			continue
+		}
 		localCredentialSpecFile := credSpecSplit[1]
-		err := cs.os.Remove(localCredentialSpecFile)
+		localCredentialSpecFilePath := filepath.Join(cs.credentialSpecResourceLocation, localCredentialSpecFile)
```

Comment:
Can you add unit test to make sure we return the correct file path so we can remove the file?

Reply:
This code is already covered by unit tests.

Reply:
In your test, when you call **mockOS.EXPECT().Remove(gomock.Any()).Return(nil).AnyTimes()**, you put gomock.Any() in it, not the real file path. That's why at the beginning, we did not catch this bug. Can you correct your unit test to make it test the right thing?

Reply:
Done.